src/main/java/io/kestra/plugin/notifications/mail/MailTemplate.java

```diff
@@ -15,6 +15,8 @@
 import java.util.Map;
 import java.util.Objects;
 
+import static io.micronaut.core.util.StringUtils.EMPTY_STRING;
```

Comment:
We try to avoid depending on Mictonaut core, please use `""` instead.

Reply:
Handled https://github.com/kestra-io/plugin-notifications/commit/d84fa92be782dbdafc72662731f539022174ba06